.ci/Jenkinsfile

```diff
@@ -86,7 +86,7 @@ pipeline {
         }
         stage('Release') {
           options { skipDefaultCheckout() }
-          when { branch 'master' }
+          when { tag "v*" }
```

Comment:
if we wanna ensure a particular semver then we might need to be more explicit,

Reply:
Yes, let's merge and improve the regex later

Thanks!